components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/reporting-overview.component.ts

```diff
@@ -144,9 +144,9 @@ export class ReportingOverviewComponent implements OnInit, OnDestroy {
 
   onDateChanged(endDate) {
     const queryParams = {...this.route.snapshot.queryParams};
-    const endDateMoment = moment(endDate);
+    const endDateMoment = moment.utc(endDate);
     if (endDate && endDateMoment.isValid()) {
-      if (moment().utc().format('YYYY-MM-DD') === endDateMoment.format('YYYY-MM-DD')) {
+      if (moment.utc().format('YYYY-MM-DD') === endDateMoment.format('YYYY-MM-DD')) {
```

Comment:
I don't believe it does, but since the documentation provides all of its examples as `moment.utc()` it seemed like something I should update. https://momentjs.com/docs/#/parsing/utc/